packages/rn-tester/RNTester-visionOS/Info.plist

```diff
@@ -7,7 +7,7 @@
 		<key>UIApplicationPreferredDefaultSceneSessionRole</key>
 		<string>UIWindowSceneSessionRoleApplication</string>
 		<key>UIApplicationSupportsMultipleScenes</key>
-		<false/>
+		<true/>
```

Comment:
Todo: check how this affects rest of the app

Reply:
on RNM, turning this on meant going through a different code path for intializing the window / scene, so the window set by RCTAppDelegate stopped showing. I guess on this fork, y'all have already gone full custom SwiftUI entry point though.